permafrost/templates/permafrost/permafrostrole_form.html

```diff
@@ -103,8 +123,12 @@ <h6 class="m-0 text-primary font-weight-bold">
                 </div>
             </div>
             <div class="form-row border-bottom-gray-300">
```

Comment:
I'm sure this is all looking good but let's double-check bootstrap 5 form row and column layout: https://getbootstrap.com/docs/5.0/forms/layout/#form-grid. Pretty sure .form-row might be deprecated shortly.